src/main/java/org/jenkins/tools/test/VersionProvider.java

```diff
@@ -0,0 +1,41 @@
+package org.jenkins.tools.test;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.jar.Attributes;
+import java.util.jar.Manifest;
+import picocli.CommandLine;
+
+public class VersionProvider implements CommandLine.IVersionProvider {
+
+    @Override
+    public String[] getVersion() throws Exception {
+        return new String[] {getPCTVersionString(), getJavaVersionString()};
+    }
+
+    @SuppressFBWarnings(
+            value = {"NP_LOAD_OF_KNOWN_NULL_VALUE", "RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE"},
+            justification =
+                    "spotbugs false poisitive due to try-with-resources / https://github.com/spotbugs/spotbugs/issues/2191")
+    private static String getPCTVersionString() {
+        try (InputStream manifestStream = VersionProvider.class.getResourceAsStream("/META-INF/MANIFEST.MF")) {
```

Comment:
uses the metadata we now build into the jar.  unrelated to the multi-module work - but was here as it was in all the builds I had for various testing.  Should have been puled out earlier.

Reply:
>  Should have been puled out earlier.

Why wasn't it?